webapp/views/wpt-results.js

```diff
@@ -368,10 +369,25 @@ class WPTResults extends AmendMetadataMixin(Pluralizer(WPTColors(WPTFlags(PathIn
     <template is="dom-if" if="[[pathIsATestFile]]">
       <div class="history">
         <template is="dom-if" if="[[!showHistory]]">
-          <paper-button id="show-history" onclick="[[showHistoryClicked()]]" raised>
-            Show history
+          <paper-button id="show-history" onclick="[[showNewHistoryClicked()]]" raised>
```

Comment:
`showNewHistoryClicked` does not need to be invoked here, and you could set `this.showNewHistory` to true in the method without returning an anonymous function.